Makefile

```diff
@@ -21,30 +24,35 @@ help: 				## show options and their descriptions
 all:  				## clean the working environment, build and test the packages, and then build the docker image
 all: clean test docker
 
-rsa: 				## create tmp/ and generate RSA keys
+tmp: 				## create ./tmp
 	if [ -d "./tmp" ]; then rm -rf ./tmp; fi
 	mkdir tmp
+
+rsa: tmp			## generate RSA keys
 	openssl genrsa -out ./tmp/id_rsa 2048
 	openssl rsa -in ./tmp/id_rsa -pubout > ./tmp/id_rsa.pub
+
+cert: rsa
 	printf '\n\n\n\n\n\n\n' | openssl req -new -x509 -sha256 -key ./tmp/id_rsa \
 		-out ./tmp/server.crt -days 3650
```

Comment:
Is creating the certificate still needed?

Reply:
It'll be needed when I add the HTTPS stuff back in for the terraform, so I'm just leaving it in.